sdk/openai/Azure.AI.OpenAI/src/Generated/AudioTranscriptionOptions.Serialization.cs

```diff
@@ -193,6 +201,59 @@ internal static AudioTranscriptionOptions DeserializeAudioTranscriptionOptions(J
                 serializedAdditionalRawData);
         }
 
+        private BinaryData SerializeMultipart(ModelReaderWriterOptions options)
```

Comment:
Why were the patterns in https://github.com/annelo-msft/azure-sdk-for-net/pull/18 not followed for this?

Reply:
This is a private function to serialize model to IPersistentModel API. There is a `ToMultipartRequestContent` to serialize model to multipart and send as content in wire.